notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,949 @@
+{
```

Comment:
<p>I have not been using the <code>df_</code>  prefix in previous lectures as i decided it is bad practice :) i would use just <code>train</code>  and <code>test</code>  or <code>train_data</code>  etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>did you also stop import numpy as np?</p><p> </p>

Reply:
<p>are you teasing me :) </p><p> </p><p>using aliases is fine, but adding data types to objects is unnecessary IMO</p>